x-pack/plugin/runtime-fields/src/main/java/org/elasticsearch/xpack/runtimefields/mapper/AbstractScriptMappedFieldType.java

```diff
@@ -149,7 +179,7 @@ public SpanQuery spanPrefixQuery(String value, SpanMultiTermQueryWrapper.SpanRew
     }
 
     private String unsupported(String query, String supported) {
-        String thisField = "[" + name() + "] which is of type [script] with runtime_type [" + runtimeType() + "]";
+        String thisField = "[" + name() + "] which is of type [runtime] with runtime_type [" + runtimeType() + "]";
```

Comment:
This is something I bumped into while working on this. Just had the old name left over.

Reply:
use the constant, then renaming is much easier.